css/style.css

```diff
@@ -0,0 +1,43 @@
+* {
```

Comment:
It is best practice to not have empty class selectors. Please remove the empty class selectors